### PR TITLE
Update integration.markdown

### PR DESCRIPTION
**Pull request by Andreas2430**
## Proposed change

Clarified under which circumstances users should opt for using the 'left' Riemann sum method rather than 'right' or 'trapezoidal' to avoid wrong calculations by the 'Riemann sum integration' integration.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: closes [#74434](https://github.com/home-assistant/core/issues/74434)

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @Andreas2430, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#23310).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_integrations/integration.markdown [hunk 1]
```diff
@@ -93,7 +93,7 @@ method:
   default: trapezoidal
 {% endconfiguration %}
 
-In case you have an appliance which produces spikey consumption (like an on/off electrical boiler) you should opt for the `left` method to get accurate readings.
+In case you expect that your source sensor will provide several subsequent values that are equal, you should opt for the `left` method to get accurate readings. Due to limitations in Home Assistant, the `right` and `trapezoidal` methods will [provide incorrect values](https://github.com/home-assistant/core/issues/74434) in such cases.
```
Comment (frenck) [inline source/_integrations/integration.markdown]: We don't document issues in our documentation; as it ends up sticking there for a long time (even after the issues are solved). Thus are hard to maintain.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/integration.markdown (hunk 1 above)
Comment (Andreas2430) [reply source/_integrations/integration.markdown]: I understand your argumentation and adapted the text.
What is a good way then to inform users about the reason why they should opt for the 'left' method in these cases? There are probably many tech-savvy users who are rather less happy with just being told to use a certain integration method in certain cases, and would like to understand why the documentation is asking them to select the 'left' method.

**Review by dgomes — APPROVED** [review]

File: source/_integrations/integration.markdown (hunk 1 above)
Comment (frenck) [reply source/_integrations/integration.markdown]: > What is a good way then to inform users about the reason why they should opt for the 'left' method in these cases?

That is a good question. I think best is to fix things 😬 Take away these limitations? Maybe you can help out with that!

**Review by frenck — APPROVED** [review]

**Pull request merged**